public/controllers/management/components/management/ruleset/actions-buttons.js

```diff
@@ -214,8 +301,25 @@ class WzRulesetActionButtons extends Component {
     // Manage files
     const manageFiles = (
       <WzButtonPermissions
-        buttonType='empty'
-        permissions={[{action: 'manager:upload_file', resource: `file:path:/etc/${section}`}]}
+        buttonType="empty"
+        permissions={[
+          {
+            action: `${((this.props || {}).clusterStatus || {}).contextConfigServer}:upload_file`,
+            resource: `file:path:/etc/${section}`,
+          },
+          {
+            action: `${((this.props || {}).clusterStatus || {}).contextConfigServer}:read`,
+            resource: `file:path:/etc/${this.props.msg}`,
+          },
+          {
+            action: `${((this.props || {}).clusterStatus || {}).contextConfigServer}:read_file`,
+            resource: `file:path:/etc/${this.props.msg}`,
```

Comment:
Should it be the `section` variable instead `this.props.msg`? I checked the code and didn't found where it is coming from